### ISSUE TITLE
Add keyboard nav to project viewer in browser

### ISSUE THREAD

**Issue by Pomax** [mozilla/webmaker-core#590]
In the browser I am used to using my cursor keys for step-wise navigation (typically "prev" vs "next" navication, but sometimes omnidirectional), and it feels like our play mode is a perfect candidate for this, too.

cursor key -> nagivate one page in that direction, rather than clicking the dpad buttons. This should be relatively little work to add to the dpad, such that every time it renders a directional tab, it also adds a document listener for `keyup` for the relevant key, unbinding it again when the tab is removed from the rendered content.

**Comment by alanmoo** [comment]
@Pomax: Just to clarify, you're saying remove the event listener during the `componentWillUnmount` phase of the lifecycle, right?

**Comment by Pomax** [comment]
yeah, the general idea is to addEventListener during `componentDidMount`, and removing it again during `componentWillUnmount`
